Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -768,11 +799,6 @@ public static void HandleFumble(Session session, params string[] parameters)
         [CommandHandler("god", AccessLevel.Sentinel, CommandHandlerFlag.RequiresWorld, 0)]
         public static void HandleGod(Session session, params string[] parameters)
         {
-            // @god - Sets your own stats to the specified level.
-
-            //TODO: output
-
-            //output: You are now a god!!!
```

Comment:
these comments should remain as the command hasn't been implemented, just the output